config/options.go

```diff
@@ -41,6 +41,7 @@ var (
 
     // Server
     WEB_PORT = MustNewIntOption("web.port", 8080, "The port for the web interface to serve on.");
+    WEB_MAX_FILE_SIZE_KB = MustNewIntOption("web.maxsizekb", 2 * 1024, "The maximum allowed file size (in KB) submitted via POST request.  The default is 2000 KB (2 MB).");
```

Comment:
2000KB is not 2MB, 2000kB is 2MB (note the casing).
Outside of networks and storage, we only use K/Ki (2^10), not k (1000).
So despite any casing, we will **always** use 2^10 as k/K (ki/Ki).